File: Desarrollo/ACU/ACU_FE/App_Comedor_UNMSM/app/src/main/java/models/Estudiante.kt
package models

class Estudiante(
    val codigo_estudiante: String,
    val correo: String,
    val password: String,
    val nombres: String,
    val apellido_pat: String,
    val apellido_mat: String,
    val foto_url: String,
    val esta_penalizado: Boolean,
    val es_jedi: Boolean
)

File: Desarrollo/ACU/ACU_BE/db/models/Estudiante.py
from sqlalchemy import Column, Integer, String, Boolean, Date, ForeignKey
from sqlalchemy.orm import relationship

from db.base_class import Base


class Estudiante(Base):
    codigo_estudiante = Column(String, primary_key=True, index=True)
    correo = Column(String, nullable=False)
    password = Column(String, nullable=False)
    nombres = Column(String, nullable=False)
    apellido_pat = Column(String, nullable=False)
    apellido_mat = Column(String, nullable=False)
    foto_url = Column(String, nullable=True)
    esta_penalizado = Column(Boolean, nullable=False)
    ticket = relationship("Ticket", back_populates="estudiante")
